drivers/flash/flash_handlers.c

```diff
@@ -36,6 +37,14 @@ static inline int z_vrfy_flash_erase(const struct device *dev, off_t offset,
 }
 #include <zephyr/syscalls/flash_erase_mrsh.c>
 
+static inline int z_vrfy_flash_get_size(const struct device *dev, uint64_t *size)
+{
+	K_OOPS(K_SYSCALL_DRIVER_FLASH(dev, get_size));
```

Comment:
If this is optional, shouldn't this be changed to:

```suggestion
	K_OOPS(K_SYSCALL_OBJ(dev, K_OBJ_DRIVER_FLASH));
```

Reply:
Taken in.